lib/main.js

```diff
@@ -79,4 +83,22 @@ module.exports = {
 
 }
 
+function _findFileInParents(path, encoding, silent) {
+  for (var i = 0; i < 100; i++) {
+    try{
+      fs.readFileSync(path, { encoding: encoding })
+      return path
+    }
+    catch (e){
```

Comment:
need to validate the exception is because file does not exist. would be better to not read file twice. check out `fs.exists`